docs/provisioning_aws_mac.md

```diff
@@ -18,6 +18,37 @@ sudo diskutil repairDisk $PDISK
 sudo diskutil apfs resizeContainer $APFSCONT 0
 ```
 
+### Setting up Xcode
+
+Download the latest versions of Xcode and Command Line Tools for Xcode from
+https://developer.apple.com/download/more/
+
+You’ll need to be logged in with your Apple ID to be able to download the
+binaries. There are big: 10+GB for both of them. Once the archives are
+downloaded install them on the host.
```

Comment:
There are articles out there on how to generate a cookie that developer.apple.com accepts, so you can download directly from the site via curl or wget. For now, this should be good enough though :+1: